src/lib/fhe/__mocks__/fhe-service.ts

```
@@ -0,0 +1,17 @@
+/**
+ * Mock implementation of the FHE service for testing
+ */
+
+export const mockFHEService = {
+  encrypt: vi.fn((data) => Promise.resolve(`encrypted-${data}`)),
```

ESLint

[error] 'vi' is not defined.

(no-undef)